layouts/default.vue

```diff
@@ -60,31 +46,15 @@ export default {
   position: relative;
 }
 
-.app-footer {
-  flex: 0 0 auto;
-}
-
-#__nuxt,
-#__layout {
-  min-height: 100vh;
-}
-
-.app-container .app-header {
-  position: sticky;
-  top: 0;
-}
+// #__nuxt,
+// #__layout {
+//   min-height: 100vh;
+// }
```

Comment:
Yes. It looks like it might have come with the scaffolded project, but I couldn't see anywhere it was used, but I couldn't be sure if we'd ever need it, so I left it.

What do you think - should I just delete it? (There are also some Nuxt demo files that could be deleted.)

Reply:
ah no harm in keeping it. I'm not sure either!

Reply:
Minor tech debt instead of minor shoot-yourself-in-the-foot, haha.